### DISCUSSION TITLE
Wireless Connectivity Solutions for ESP32-P4 · Espressif Developer Portal

### DISCUSSION THREAD

**Discussion by giscus** [espressif/developer-portal#225] [General]
# Wireless Connectivity Solutions for ESP32-P4 · Espressif Developer Portal

The developer resources in just one place!

https://developer.espressif.com/blog/wireless-connectivity-solutions-for-esp32-p4/

**Comment by georgik** [comment]
Simple example which demonstrates how to use `ESP-hosted` with [`esp_wifi_remote` component](https://components.espressif.com/components/espressif/esp_wifi_remote) is available on [GitHub](https://github.com/georgik/esp32-swift-weather-display/tree/main).
